docs/pages/contributor_guidelines.rst

```diff
@@ -41,3 +41,32 @@ Documentation
   'reddit.com'. This helps prevent any confusion between what is universally
   true between all reddits and what specifically applies to the most known
   instance.
+
+EOL Issues
+----------
+
+Depending on the OS you are editing the source on, you may run into one of two
+strange errors. Firstly, PRAW was written on a Unix-based OS. This means, that
+the source's ``EOLs`` (End Of Lines) are the Unix ``LF`` character, a.k.a "\\n"
+a.k.a hex ``0x0A``.
+
+On Windows, ``EOLs`` are ``CRLF``, which is ``0x0D0A`` ("\\r\\n") or if you are
+on a Mac with OS-9 and earlier ``EOLs`` are ``0x0D`` ("\\r"). More info on
+``EOLs`` can be found `here <http://bit.ly/1Ug8Xyo>`_.
+
+This can potentially pose one of two problems.
+
+1. Your ``EOLs`` may not be automatically converted and your code will break PRAW.
+
+2. Your editor automatically converts `EOLs`, at the cost that three invisible
+   hexadecimal values are added at the beginning of a line, which also breaks
+   PRAW. This can be determined to be the case if in the commit history on Github,
+   the a line of a file is removed and re-added with no visible change.
+
+The solutions to each are as follows:
```

Comment:
I know that git has built in support for converting line endings. Do you have an example commit that shows the line endings actually being messed up? This is the first time I've encountered anyone with such a issue and I suspect there is an easier way than manually editing files in hexeditors. For instance, sublime text (my editor) will simply allow you to change line ending mode. Even if you don't use it as your editor, surely that's simpler than manually using a text editor. There are other command-line tools to do the same thing.